infra/tpu-pytorch-releases/artifacts.auto.tfvars

```diff
@@ -35,39 +35,39 @@ xrt_versioned_builds = [
   {
     accelerator    = "tpu"
     python_version = "3.10"
-    pytorch_git_rev = "v2.1.0-rc5"
-    package_version = "2.1.0rc5+xrt"
+    pytorch_git_rev = "v2.1.0-rc6"
+    package_version = "2.1.0"
   },
   {
     accelerator  = "cuda"
     python_version = "3.10"
     cuda_version = "12.0"
-    pytorch_git_rev = "v2.1.0-rc5"
-    package_version = "2.1.0rc5+xrt"
+    pytorch_git_rev = "v2.1.0-rc6"
+    package_version = "2.1.0"
```

Comment:
Can you add back the `+xrt` extra? We want to distinguish between the main build and the XRT one.

Reply:
Thanks, updated